.github/workflows/pom_update.yml

```diff
@@ -26,12 +26,20 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
         git checkout -b $BRANCH
+        # Get the current version
+        CURR_VERS=$(./mvnw help:evaluate -Dexpression=project.version -q -DforceStdout | sed 's/-SNAPSHOT$//')
```

Comment:
Maybe use the normal full name `CURR_VERSION` instead of `CURR_VERS`?

Reply:
Same, that's also been changed already